app/models/asset_manager.rb

```diff
@@ -78,4 +78,8 @@ def questions
     end
     questions
   end
+
+  def get_scope
```

Comment:
This is great! Can we change the `<user type>_initial_state` methods in `application_helper.rb` to call these helper methods for scope instead so there's no repeated code?